lib/Demangling/Remangler.cpp

```diff
@@ -1019,7 +1019,20 @@ void Remangler::mangleFunctionSignatureSpecializationParam(Node *node) {
       Buffer << 'x';
       return;
     default:
-      if (kindValue & unsigned(FunctionSigSpecializationParamKind::Dead)) {
+      if (kindValue &
+          unsigned(
+              FunctionSigSpecializationParamKind::ExistentialToGeneric)) {
+        Buffer << 'e';
```

Comment:
It seems weird that this is an if/else if construct that duplicates some of the cases here. Can you instead redo this as a series of independent ifs instead?

Reply:
Ok. It seems strange that the first letter is lower case and the rest are upper case, for each of the different argument kinds. But I guess that’s what we have today already.